source-map-support.js

```diff
@@ -381,14 +381,16 @@ function wrapCallSite(frame) {
 }
 
 // This function is part of the V8 stack trace API, for more info see:
-// http://code.google.com/p/v8/wiki/JavaScriptStackTraceApi
+// https://github.com/v8/v8/wiki/Stack-Trace-API
 function prepareStackTrace(error, stack) {
   if (emptyCacheBetweenOperations) {
     fileContentsCache = {};
     sourceMapCache = {};
   }
 
-  return error + stack.map(function(frame) {
+  var message = (error instanceof Error) ? error : error.message;
```

Comment:
To match `console.trace` I think that this should be something like `` `${error.name}: ${error.message}` `` instead of just `error.message`, what do you think?